docs/gallery/interactive/pyfar_coordinates.ipynb

```diff
@@ -0,0 +1,432 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Coordinates\n",
+    "\n",
+    "In acoustics research, we often deal with different coordinate systems when handling sampling points, for example, microphone positions in a spherical microphone array or loudspeaker positions in a sound field synthesis system. This can get complicated. That's why we have the [pyfar.Coordinates](https://pyfar.readthedocs.io/en/stable/classes/pyfar.coordinates.html) . The class was designed for storing, working with, and accessing coordinate points. You can easily switch between different coordinate systems, rotate points, and do other useful operations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "%matplotlib inline"
```

Comment:
```suggestion
```

Reply:
if i dont use it, i need to use plt.show() to show all the plots...

Reply:
Is this environment dependent? Works without `%matplotlib inline` in my case (VScode, python 3.11.7)

Reply:
Found this on the web: https://stackoverflow.com/questions/65934740/is-matplotlib-inline-still-needed